2022q2/valgrind.adoc

```diff
@@ -32,17 +36,22 @@ pointers in ECX.
 * improve the setting of errno by C allocating functions
 * fix building of Valgrind with llvm-devel (15.0.0)
 
-
 For FreeBSD 13.1 / 14.0 there are
+
 * syscall wrappers for funlinkat, copy_file_range, swapoff, shm_open2
 * add K_INFO handling to fcntl
 * add handling for new auxv entries
 * added some default suppressions for DRD and Helgrind
 
+There is now an initial version of vgdb invoker support - this allows
+vgdb to use ptrace to force valgrind to valgrind poll for gdb commands.
+This is not yet available in the ports versions.
+
 That does not leave much in the way of outstanding issues. I expect that
 14.0 and newer versions of llvm will keep on requiring support. Apart
 from that there is
 
-* vgdb invoker support
+* some small problems with error messages getting the correct source
+information
 * better core dumps (low priority)
 * TLS handling for Helgrind (difficult if not impossible)
```

Comment:
Missed that the first time, but perhaps expand TLS (thread-local storage, right?) so people for whom it means Transport Layer Security aren't brought up short?

Reply:
Agreed again.